lms/views/api/groups.py

```diff
@@ -10,6 +10,7 @@
     route_name="api.courses.group_sets.list",
 )
 def course_group_sets(_context, request):
-    return request.product.plugin.grouping.get_group_sets(
+    course = request.find_service(name="course").get_by_context_id(
         request.matchdict["course_id"]
     )
```

Comment:
Course can be `None` here, which I think is an error in all cases. If it is, then the result is a `None` has no attribute `lms_id` type of error, which is unhandled.

I think we have to deal with this here?